### PR TITLE
Homepage structure and more assets

### PR DESCRIPTION
**Pull request by htmelvis**
I have added more structure to the homepage and also included fonts, extended the base layout and made some minor tweaks

screen shot 2015-07-14 at 11 56 48 pm

### CODE REVIEW

**Comment by matsilva** [comment]
Assigning to myself to review React components, then will hand off to @machuga most likely.

File: app/assets/javascripts/components/on-boarding/terminal-prompt.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @jsx React.DOM
+ */
+
+var React = require('react');
+
+var TerminalPrompt = React.createClass({
+
```
Comment (matsilva) [inline app/assets/javascripts/components/on-boarding/terminal-prompt.js:8]: Mind implementing prop types? https://facebook.github.io/react/docs/reusable-components.html
Something like:

```
propTypes: {
    update: React.PropTypes.func.isRequired // can drop isRequired if it is optional
},
```

**Comment by htmelvis** [comment]
That would be excellent...I was thinking on how to validate and there we go! to the rescue!

will do

File: app/assets/javascripts/components/on-boarding/terminal-window.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      case 38: // Up
+      case 39: // Right
+      case 40: // Down
+        break;
+      default:
+        this.moveCursor()
+    }
+  },
+  update: function(e){
+    //TODO: Need real live validation here for emails
```
Comment (matsilva) [inline app/assets/javascripts/components/on-boarding/terminal-window.js:35]: When merge this, lets make sure to get this todo into another issue. Could be something we can handoff to someone else?

**Comment by htmelvis** [comment]
The Cursor TODO?

File: app/assets/javascripts/components/on-boarding/terminal-window.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    if(e.target.value === 'ed@htmelvis.com'){
+      var msg = "What's up Ed!";
+    } else{
+      var msg = "OOOPpppss you seem to have not typed in an email address";
+    }
+
+    this.setState({
+      message: msg
```
Comment (matsilva) [inline app/assets/javascripts/components/on-boarding/terminal-window.js]: For naming purposes and understanding, would it make sense to rename this property to `terminalMessage` ?

File: app/assets/javascripts/components/on-boarding/terminal-window.js (hunk 2 above)
Comment (htmelvis) [inline app/assets/javascripts/components/on-boarding/terminal-window.js:35]: Ok. Good Idea. See Issue #20

File: app/assets/javascripts/components/on-boarding/terminal-window.js (hunk 3 above)
Comment (htmelvis) [inline app/assets/javascripts/components/on-boarding/terminal-window.js]: terminalMessage makes more sense for sure.

**Comment by matsilva** [comment]
Overall nice job on the React Components and getting this initial barebones setup. Mind checking out the comments I left ?

I think a good thing to mention in the description of this PR is we will need to create and additional ticket to handle:
- Capturing & user data.
- Real time validation on data(check if user !exist && email is valid)
- Sending data to server
- Confirmation or some other view to follow up with.

File: app/assets/javascripts/components/on-boarding/terminal-prompt.js (hunk 1 above)
Comment (matsilva) [inline app/assets/javascripts/components/on-boarding/terminal-prompt.js:8]: Keep in mind propTypes are purely for developers. Not for user validation ;)

**Comment by matsilva** [comment]
+1 on the design, looks freakin awesome.

File: app/views/home/index.html.haml [hunk 4]
```diff
@@ -7,18 +7,46 @@
...
+            -#%h3= meetup.name
+            -#%h4= meetup.starts_at
+
+
+    .row.icon-row
+      -#TODO: NEED SOME GRAPHICS FOR THESE GUYS!
+      .third
+        %figure.upcoming-meetups
+          = link_to root_path do
+            =image_tag 'brand/circlebraces.png', :class => 'brandmark'
```
Comment (machuga) [inline app/views/home/index.html.haml]: When using symbols we can shorthand to:

``` haml
= image_tag 'brand/circlebraces.png', class: 'brandmark'
```

File: app/views/layouts/application.html.haml [hunk 5]
```diff
@@ -18,26 +18,29 @@
...
+                    %li= link_to 'Logout', destroy_user_session_path, 'data-method' => 'delete'
+                - else
+                  %ul
+                    %li= link_to 'Login', new_user_session_path
+                    %li= link_to 'Sign Up', new_user_registration_path
 
       -#%h3= t(:tag_line)
       -#%p Write Code? Live around Erie? Come join us!
-    %main#site-main
+    %section#site-main
```
Comment (machuga) [inline app/views/layouts/application.html.haml]: Any reason you don't want to use the HTML5 main tag?

**Comment by machuga** [comment]
Minor question.  Looks fantastic!

**Comment by htmelvis** [comment]
I honestly don't remember doing that. Must have been subliminal! No reason, besides that it was like past 1am haha.

**Comment by htmelvis** [comment]
In my newest commit I will solve the questions and so on and make adjustments

File: app/assets/javascripts/components/on-boarding/terminal-prompt.js (hunk 1 above)
Comment (htmelvis) [inline app/assets/javascripts/components/on-boarding/terminal-prompt.js:8]: totally see that now...awesome thats a nice feature

**Comment by htmelvis** [comment]
screen shot 2015-07-15 at 11 29 23 pm

Also added icons to the bottom row with gif and png versions and a script to animate them

**Comment by htmelvis** [comment]
screen shot 2015-07-16 at 12 24 34 am

Basic layout for login pages included in commit [#4b2b](https://github.com/htmelvis/coderie-site/commit/4b2b6c3986fd0b25c54699ec055da93447fb0b07)

**Comment by lxcodes** [comment]
Awesome graphics in here :+1:

**Comment by machuga** [comment]
I fixed the merge conflicts in the JS file - all merged up!

**Comment by machuga** [comment]
This looks absolutely fantastic, btw.  And the graphics are awesome!

**Comment by lxcodes** [comment]
:+1: :cake: :tada:

**Pull request merged**